Gemfile

```diff
@@ -1,6 +1,7 @@
 source 'https://rubygems.org'
 
-gem 'jekyll', '~> 3.1.6'
-gem 'github-pages', '84'
+gem 'jekyll'
+gem 'github-pages'
 gem 'geocoder'
 gem 'lolize', :require => 'lolize/auto'
+ruby '~> 2.1.5'
```

Comment:
A suggestion: replace this line with `ruby File.read(".ruby-version")`.
This way, we'd only need to update the ruby version in the `.ruby-version` file, and the Gemfile would follow. :)

Reply:
ah and also:
while I don't actually know the reason for it (or *if* there is a reason for it) the convention in a Gemfile is usually to put the Ruby version right under the `source 'https://rubygems.org'` line. Could you move it up there?